sirepo/template/radia.py

```diff
@@ -27,11 +28,59 @@
 import sirepo.util
 import time
 
+class DataRequester(template_common.DataRequesterBase):
+    """Fetch data from the template either for download or to use on the client
+
+
+    """
+
+    def __init__(self):
+        """
+        Args:
+            xxx (xxx):
+           XXX
+        """
+        self.appExporter = Exporter()
+
+    def export(self, sim):
+        """Export to file"""
+        return self.exporter.export(sim)
+
+    def fetch(self, sim_id, sim_type):
```

Comment:
what is this doing that the superclass isn't.

Reply:
Nothing yet, but all the sim-specific data gathering when the time comes